lib/strong_migrations.rb

```diff
@@ -63,8 +63,16 @@ def self.lock_timeout_limit
     @lock_timeout_limit
   end
 
-  def self.add_check(&block)
-    checks << block
+  def self.add_check(only_action = nil, start_after: nil, &block)
```

Comment:
The `&block` already accepts operation name as an argument, so I think the implementation can be revised and simplified. 
For example, to something like this
https://github.com/fatkodima/online_migrations/blob/5384767ccd3f8246b123780525d9a9763251ac38/lib/online_migrations/config.rb#L207-L260

Reply:
Yes, true that. Although, I believe the argument for `add_check` conveys the information a bit better than first line in a check body that says which action this check is about. Even though the code do exactly the same, but implicitly. 

```ruby
StrongMigrations.add_check(:remove_index) do 
  ...
end
```

vs

```ruby
StrongMigrations.add_check do |action, args|
  next unless action == :remove_index

  ...
end

But, yeah, if you really don't like it, I can remove, my main focus was on `start_after`.